e2e/js_package/.npmrc

```diff
@@ -0,0 +1,2 @@
+node-linker=pnp
```

Comment:
any comments about how this file participates in the example? do we read it? does npm read it?